src/globals.css

```
@@ -16,9 +16,9 @@
   --color-affirm: #3f6144;
 
   --font-sans:
-    "Inter", -apple-system, BlinkMacSystemFont, "Hiragino Kaku Gothic ProN", Meiryo, sans-serif;
-  --font-serif: "Crimson Pro", Garamond, Georgia, serif;
-  --font-mono: "JetBrains Mono", "Monaco", "Menlo", monospace;
+    'Inter', -apple-system, BlinkMacSystemFont, 'Hiragino Kaku Gothic ProN', Meiryo, sans-serif;
```

Stylelint (17.14.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)